src/DocumentFormat.OpenXml/Packaging/OpenXmlSupportedRelationshipExtensions.cs

```diff
@@ -0,0 +1,435 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using DocumentFormat.OpenXml.Features;
+using System;
+
+namespace DocumentFormat.OpenXml.Packaging
```

Comment:
nit: can you do a namespace scoped file? i.e. add a semicolon to the end here and remove the tab/braces